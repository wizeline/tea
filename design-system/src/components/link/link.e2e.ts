import { newE2EPage } from '@stencil/core/testing';

describe('wz-link', () => {
  it('should render a link-component', async () => {
    const page = await newE2EPage();
    await page.setContent('<wz-link>Hello, my name is Marty McFly</wz-link>');
    const link = await page.find('wz-link');
    expect(link.innerText).toMatchSnapshot();
  });
});
