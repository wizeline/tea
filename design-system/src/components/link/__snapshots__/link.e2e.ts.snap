// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-link should render a link-component 1`] = `"Hello, my name is Marty McFly"`;
